### PR TITLE
Add more fields to desire lrp body for OPI

### PR DESCRIPTION
**Pull request by akshaymankar**
[#168003564]

Signed-off-by: Mario Nitchev <marionitchev@gmail.com>

Thanks for contributing to cloud_controller_ng. To speed up the process of reviewing your pull request please provide us with:

* A short explanation of the proposed change:

This change adds more fields to desire LRP request for eirini.

* An explanation of the use cases your change solves

These fields will in general be useful for writing extensions to eirini. This also allows eirini team to not send multiple pull requests for every time we implement new features.

* Links to any other associated PRs

N/A

* [x] I have reviewed the [contributing guide](https://github.com/cloudfoundry/cloud_controller_ng/blob/master/CONTRIBUTING.md)

* [x] I have viewed, signed, and submitted the Contributor License Agreement

* [x] I have made this pull request to the `master` branch

* [x] I have run all the unit tests using `bundle exec rake`

* [x] I have run [CF Acceptance Tests](https://github.com/cloudfoundry/cloud_controller_ng#cf-acceptance-tests-cats)

### CODE REVIEW

**Comment by tcdowney** [comment]
If y'all merge this you can close https://github.com/cloudfoundry/cloud_controller_ng/pull/1463!

**Comment by cwlbraa** [comment]
hey, I don't know what the deal is with the new CLA bot, but it says y'all haven't signed. Can you fix that? I've dragged this near the top of our backlog.

**Comment by jpalmerLinuxFoundation** [comment]
@akshaymankar, what happens when you click on the link? 
image

**Comment by akshaymankar** [comment]
I get this:

Screen Shot 2019-10-18 at 8 21 55 pm

**Comment by jpalmerLinuxFoundation** [comment]
Please close and reopen the PR

**Comment by jpalmerLinuxFoundation** [comment]
Please attempt a close/reopen again.

**Comment by julz** [comment]
bump because this is blocking us (sorry for all the CLA fun :))

**Comment by jpalmerLinuxFoundation** [comment]
> bump because this is blocking us (sorry for all the CLA fun :))

@julz, @akshaymankar, it looks like EasyCLA check is passing.
image
Note that the "CLA Not Signed" comment will not update.

File: lib/cloud_controller/opi/apps_client.rb [hunk 1]
```diff
@@ -96,7 +96,8 @@ def to_hash
           buildpack_lifecycle: {
             start_command: @process.specified_or_detected_command,
             droplet_hash: @process.desired_droplet.droplet_hash,
-            droplet_guid: @process.desired_droplet.guid
+            droplet_guid: @process.desired_droplet.guid,
+            # stack: @process.app.lifecycle_data.stack
```
Comment (sethboyles) [inline lib/cloud_controller/opi/apps_client.rb]: got a stray comment here

Comment (akshaymankar) [reply lib/cloud_controller/opi/apps_client.rb]:
```suggestion
```

**Pull request merged**